README.md

```diff
@@ -51,10 +51,10 @@ cd metaphysics
 
 # Run the setup script
 
-source ./scripts/setup.sh
+yarn setup
```

Comment:
The setup script is responsible for installing `yarn`, so I don't think we should recommend running it this way. (I realize practically everyone already has `yarn` installed, but the setup script aims to work for a brand new developer and environment rather than make those assumptions.)

Reply:
Ah, super good point 👍 Will revert